### ISSUE TITLE
kubeshark:51.0.39 reports version 0.0.0

### ISSUE THREAD

**Issue by andrewrothstein** [kubeshark/kubeshark#1468]
**Describe the bug**
kubeshark:51.0.39 reports version 0.0.0

**To Reproduce**
```
arothste@console:~$ /usr/local/kubeshark_51.0.39_linux_amd64/kubeshark version
0.0.0
```

**Expected behavior**
```
arothste@console:~$ /usr/local/kubeshark_51.0.39_linux_amd64/kubeshark version
51.0.39
```

**Comment by andrewrothstein** [comment]
same for the published v52.0.0

**Comment by corest** [comment]
Hey @andrewrothstein ! Sorry for such a late response. The issue was fixed here https://github.com/kubeshark/kubeshark/pull/1504/files . Thanks for reporting!
